src/screens/Questions.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useState } from 'react';
+import { View } from 'react-native';
+import { ProgressSteps, ProgressStep } from 'react-native-progress-steps';
+import DescriptiveQ from '../components/DescriptiveQ';
+import TimesQ from '../components/TimesQ';
+
+const questions = [
+  [
+    {
+      Question: 'Question1',
+      Answer: [
+        { label: 'Option1' },
+        { label: 'Option2' },
```

Comment:
3 options 1 other

Reply:
do you also want a text box to show up when they select the other

Reply:
I mean that's the whole point